github.go

```diff
@@ -66,19 +65,13 @@ func NewGithubClient(
 	token string,
 	releaseFilter ReleaseFilter,
 	assetFilter AssetFilter,
-	connectionTimeout time.Duration,
```

Comment:
with this change we should either bump major version as it breaks BC, or store this and create timeout context internally - not sure what is better in this case

Reply:
I will make some changes to keep compatibility for now :+1:  
but we might need to do break this in the future, currently it accepts no token but Github V4 API [requires the requests to be authenticated](https://docs.github.com/en/graphql/guides/forming-calls-with-graphql#authenticating-with-graphql) to perform GraphQL operation

Reply:
Create a ticket not to forget about it. OR maybe do it right away